pkg/fanal/analyzer/pkg/bottlerocket_inventory/bottlerocket_inventory.go

```diff
@@ -0,0 +1,105 @@
+package bottlerocket_inventory
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+	"slices"
+	"time"
+
+	"github.com/aquasecurity/trivy/pkg/fanal/analyzer"
+	"github.com/aquasecurity/trivy/pkg/fanal/types"
+	"github.com/aquasecurity/trivy/pkg/log"
+)
+
+func init() {
+	analyzer.RegisterAnalyzer(newBottlerocketInventoryAnalyzer())
+}
+
+const analyzerVersion = 1
+
+var requiredFiles = []string{
+	"aarch64-bottlerocket-linux-gnu/sys-root/usr/share/bottlerocket/application-inventory.json",
+	"x86_64-bottlerocket-linux-gnu/sys-root/usr/share/bottlerocket/application-inventory.json",
+}
+
+type bottlerocketInventoryAnalyzer struct{}
+
+func newBottlerocketInventoryAnalyzer() *bottlerocketInventoryAnalyzer {
+	return &bottlerocketInventoryAnalyzer{}
+}
+
+type ApplicationInventory struct {
+	Content []struct {
+		Name            string    `json:"Name"`
+		Publisher       string    `json:"Publisher"`
+		Version         string    `json:"Version"`
+		Release         string    `json:"Release"`
+		InstalledTime   time.Time `json:"InstalledTime"`
+		ApplicationType string    `json:"ApplicationType"`
+		Architecture    string    `json:"Architecture"`
+		URL             string    `json:"Url"`
+		Summary         string    `json:"Summary"`
+	} `json:"Content"`
+}
+
+func (a bottlerocketInventoryAnalyzer) Analyze(ctx context.Context, input analyzer.AnalysisInput) (*analyzer.AnalysisResult, error) {
+	ctx = log.WithContextPrefix(ctx, "application-inventory")
+	parsedInventorys, err := a.parseApplicationInventory(ctx, input.Content)
+	if err != nil {
+		return nil, err
+	}
+
+	return &analyzer.AnalysisResult{
+		PackageInfos: []types.PackageInfo{
+			{
+				FilePath: input.FilePath,
+				Packages: parsedInventorys,
+			},
+		},
+	}, nil
+}
+
+func (a bottlerocketInventoryAnalyzer) parseApplicationInventory(ctx context.Context, r io.Reader) ([]types.Package, error) {
+	var pkgs []types.Package
+
+	var applicationInventory ApplicationInventory
+	if err := json.NewDecoder(r).Decode(&applicationInventory); err != nil {
+		return nil, err
+	}
+
+	for _, app := range applicationInventory.Content {
+		pkg := types.Package{
+			Arch:    app.Architecture,
+			Name:    app.Name,
+			Version: app.Version,
+		}
```

Comment:
We also need epoch. Otherwise, it leads to false detection.

```
    {
      "Name": "xfscli",
      "Publisher": "bottlerocket-core-kit",
      "Version": "0.0",
      "Release": "1.1745352352.c92146c5.br1",
      "Epoch": "1",
      "InstalledTime": "2025-04-23T00:26:38Z",
      "ApplicationType": "Unspecified",
      "Architecture": "x86_64",
      "Url": "https://github.com/bottlerocket-os/bottlerocket",
      "Summary": "XFS progs cli"
    },
```